### PR TITLE
Server setup

### PR DESCRIPTION
**Pull request by git-voo**
- Setup `server.js` to handle routes and DB connection
- Setup mongoDB connection (Compass)
- Add model and controller for `item `listing and management

### CODE REVIEW

**Comment by git-voo** [comment]
[Backend + DB Setup](https://trello.com/c/0JFrgx1v/36-backend-db-setup)

**Review by shagunsharmaslu — APPROVED** [review]
lgtm

File: backend/DB/connectDB.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    try {
+      await mongoose.connect(MONGODB_URL, {
+        useNewUrlParser: true,
+        useUnifiedTopology: true,
+        family: 4
+      })
+  
+      console.log('Connected to MongoDB')
+    } catch (error) {
+      console.error('Error connecting to MongoDB:', error)
```
Comment (mohamdlog) [inline backend/DB/connectDB.js:15]: Good error catch

**Review by mohamdlog — APPROVED** [review]
lgtm;

File: backend/controllers/itemController.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+// Delete an item
+export const deleteItem = async (req, res) => {
+    try {
+        const deletedItem = await ItemModel.findByIdAndDelete(req.params.id)
+        if (!deletedItem) return res.status(404).json({ message: 'Item not found' })
+        res.status(200).json({ message: 'Item deleted successfully' })
+    } catch (error) {
+        res.status(500).json({ message: error.message })
+    }
```
Comment (nlp-saiteja) [inline backend/controllers/itemController.js:60]: The item routes are defined clearly, covering all necessary CRUD operations (getItems, getItemById, createItem, updateItem, and deleteItem)

**Review by nlp-saiteja — APPROVED** [review]
Lgtm!

**Pull request merged**